vscode/package.json

```diff
@@ -43,8 +66,47 @@
           "command": "qsharp-vscode.runEditorContents",
           "when": "resourceLangId == qsharp"
         }
+      ],
+      "view/title": [
+        {
+          "command": "qsharp-vscode.workspacesRefresh",
```

Comment:
should we update the command prefixes to match the package ID? I'm not sure on the convention here.

Reply:
Thought about that... but we'll be publishing to two different packages with the dev and stable channels, so which one? :-)

I think that's a convention rather than a requirement. Unless someone else publishes a package called 'qsharp-vscode' (and follows the convention), it shouldn't be an issue.